src/formatters/retrieveResultFormatter.ts

```diff
@@ -80,9 +80,9 @@ export class RetrieveResultFormatter implements Formatter<RetrieveResultJson> {
         name: { header: 'Package Name' },
         fullPath: { header: 'Converted Location' },
       };
-      const title = 'Retrieved Packages';
-      const options = { title: tableHeader(title), 'no-truncate': true };
+      const options = { title: tableHeader('Retrieved Packages'), 'no-truncate': true };
       this.ux.log();
+      this.ux.warn('Metadata from retrieved packages is meant for reference only, not development');
```

Comment:
@jshackell-sfdc - what do you think about this warning message displayed in command output when the flag is used?

Reply:
Nice!  Here's a slight edit - whaddya think?

'Metadata from retrieved packages is meant for your reference only, not development.'